### PR TITLE
Subdir portals agp

### PR DESCRIPTION
**Pull request by antgonza**
:dog:

### CODE REVIEW

**Comment by josenavas** [comment]
:+1:

File: qiita_pet/templates/analysis_results.html [hunk 1]
```diff
@@ -69,7 +69,7 @@ <h3 class="panel-title"><a data-toggle="collapse" data-parent="#accordion" href=
                   <h3 style="color:red">ERROR</h3>
                 {% end %}
                 {%for result, result_name in results%}
-                  <a href='{{"/results/%s" % result}}' target="resframe">{{result_name}}</a><br />
+                  <a href='{% raw qiita_config.portal_dir %}{{"/results/%s" % result}}' target="resframe">{{result_name}}</a><br />
```
Comment (squirrelo) [inline qiita_pet/templates/analysis_results.html]: this can become one string formatting call.

Comment (squirrelo) [inline qiita_pet/templates/analysis_results.html]: After discussion, this will work as is.

**Comment by squirrelo** [comment]
:+1:

**Pull request merged**